packages/e2e-test-kit/src/project-runner.ts

```diff
@@ -183,7 +183,33 @@ export class ProjectRunner {
             });
         });
     }
-
+    public waitForRecompile() {
+        let done = false;
+        return new Promise<void>((res) => {
+            this.compiler?.hooks.afterDone.tap('waitForRecompile', () => {
```

Comment:
inifinite hooks in watch mode?

Reply:
no no no. only in watch mode in tests and only if you activate it!!!